api/client/samples/analysis_kits/stocks_to_use_price_model/requirements.txt

```diff
@@ -1,4 +1,4 @@
-pandas~=0.24.2
-matplotlib~=3.1.1
-scikit-learn~=0.21.2
-jupyter
\ No newline at end of file
+pandas
```

Comment:
Summary of offline discussion: I originally removed the version restriction because I had trouble installing these dependencies locally (I think that version of Pandas required building numpy locally, which failed?). I figured it was fine to remove, since almost none of the other sample notebook requirements.txt files have version restrictions.

It may be good to at least specify a major version to avoid breakages from new library versions, though there it also a benefit to not have version restrictions (for tested notebooks) since we'll be able to keep testing that our code works with the latest library versions.